### DISCUSSION TITLE
Welcome to CuraSettingsWriter Discussions!

### DISCUSSION THREAD

**Discussion by 5axes** [5axes/CuraSettingsWriter#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BathogTech** [comment]
Excellent plugin for Cura. Only issue I have is that it prints out settings for all 8 extruders, even though my machine profile only has one.
Therefore I have to do some modification to the HTML file, before I can use it. 
Is there a way to have it only print settings for the number of extruders fitted?

**Reply by 5axes** [reply]
Looks like an issue with your printer definition. If you have 8 exturders output, it's because there are 8 instance declare in the json file. What is your machine ?

**Comment by BathogTech** [comment]
5axes Thanks for replying. My machine is an Anet A8. I will check the json definition.Your add on will be extremely useful to me as my project is a large 3D printed RC model aircraft, that has 119 printed parts. I am uploading all the parts to Thingiverse so having a way of extracting the print settings from Cura is a massive time saver. Thanks Graham Sent from Mail for Windows 10 From: 5axesSent: 31 July 2021 07:48To: 5axes/CuraSettingsWriterCc: BathogTech; CommentSubject: Re: [5axes/CuraSettingsWriter] Welcome to CuraSettingsWriter Discussions! (#6) Looks like an issue with your printer definition. If you have 8 exturders output, it's because there are 8 instance declare in the json file. What is your machine ?—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Reply by 5axes** [reply]
I have just make a short test with an Anet A8 definition and the HTML is correct ( just one extruder output in the HTML file) . Are you using the Anet A8 provided with Cura or an older or personnal printer definition ?
